### PR TITLE
Fixed bugs and removed .zip file

### PR DESCRIPTION
**Pull request by TREN-D**

### CODE REVIEW

File: InternetHospitalWebUI/src/app/Models/Filter.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    isWithParams: boolean;
+  
+    public CheckIfPropertyExist():void {
+      if (this.searchKey || this.selectedSpecialization) {
+        this.isWithParams = true;
+      }
+      else {
+        this.isWithParams = false;
+      }
+    }
```
Comment (SergiyKozachok) [inline InternetHospitalWebUI/src/app/Models/Filter.ts:8]: Is it possible to better use the ternary operator here? There will be only one line

Comment (voznovych) [reply InternetHospitalWebUI/src/app/Models/Filter.ts:8]: Moreover just this.isWithParams = this.searchKey || this.SelectedSpecialization;

Comment (TREN-D) [reply InternetHospitalWebUI/src/app/Models/Filter.ts:8]: 'this.isWithParams = this.searchKey || this.selectedSpecialization', this operation trying to set string value into this.isWithParams

Comment (TREN-D) [reply InternetHospitalWebUI/src/app/Models/Filter.ts:8]: Changed signature like Serhii said

**Review by SergiyKozachok — APPROVED** [review]

**Review by eugent482 — APPROVED** [review]

**Review by serhiidotsyk — APPROVED** [review]

**Pull request merged**